src/util/LoadPixmapLazy.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "common/Aliases.hpp"
+
+#include <QPixmap>
+
+namespace chatterino {
+void loadPixmapFromUrlLazy(const Url &url,
+                           std::function<void(QPixmap)> &&callback);
```

Comment:
- This function should be called `loadPixmapFromUrl`. There's nothing lazy about it.
- The documentation should mention that `callback` is invoked concurrently.
- The whitespace in this file should be adjusted.